panel/pane.py

```diff
@@ -560,6 +560,74 @@ def _img(self):
         return b.getvalue()
 
 
+def latex_to_img(text, size=25):
+    """
+    Returns PIL image for LaTeX equation text, using matplotlib's rendering.
+    Usage: latex_to_img(r'$\frac(x}{y^2}$')
+    From https://stackoverflow.com/questions/1381741.
+    """
+    import matplotlib.pyplot as plt
+    from PIL import Image, ImageChops
+    import io
+    
+    buf = io.BytesIO()
+    plt.rc('text', usetex=True)
+    plt.rc('font', family='serif')
```

Comment:
Should not modify rcparams globally use, should instead use:

```python
with plt.rc_context({'text.usetex': True, 'font.family': 'serif'}):
    ...
```

Reply:
Thanks; fixed.